packages/mdc-radio/index.ts

```diff
@@ -22,106 +22,97 @@
  */
 
 import MDCComponent from '@material/base/component';
-/* eslint-disable no-unused-vars */
-import {MDCSelectionControlState, MDCSelectionControl} from '@material/selection-control/index';
-/* eslint-enable no-unused-vars */
-import MDCRadioFoundation from './foundation';
+import {EventType, SpecificEventListener} from '@material/dom/index';
```

Comment:
These types have been moved to `@material/base/types`

Reply:
ah sorry - i moved them back in list PR...will revert

Reply:
We'll do this in PR #4358.